controllers/memberoperatorconfig/mapper_test.go

```diff
@@ -0,0 +1,99 @@
+package memberoperatorconfig
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	testconfig "github.com/codeready-toolchain/toolchain-common/pkg/test/config"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func TestSecretToMemberOperatorConfigMapper(t *testing.T) {
+	// when
+	secretData := map[string][]byte{
+		"che-admin-username": []byte("cheadmin"),
+		"che-admin-password": []byte("password"),
+	}
+	secret := newSecret("test-secret", secretData)
+	config := NewMemberOperatorConfigWithReset(t, testconfig.MemberStatus().RefreshPeriod("10s"))
```

Comment:
As I mentioned above we don't need the config to be present for the mapper. So you can drop this.

Reply:
Removed it with the previous commit: cc1eb37ab554057e49588fa4b56fc1bbcda08a8f